py/desitarget/fortran/README

```diff
@@ -0,0 +1,17 @@
+The files in this directory are:
+
+ADM-v1dump.f: FORTRAN code to parse UCAC/URAT files, adapted from
+              http://cdsarc.u-strasbg.fr/ftp/I/329/URAT1/access/v1dump.f
+              by Adam D. Myers (University of Wyoming).
+
+v1sub.f: Routines to help parse UCAC/URAT files, taken directly from
+         http://cdsarc.u-strasbg.fr/ftp/I/329/URAT1/access/v1sub.f
+
+v1dump: An executable compiled at NERSC via:
+
+        gfortran -o v1dump ADM-v1dump.f v1sub.f
```

Comment:
Thanks for including specific instructions for compiling v1dump.  Please also include documentation somewhere about how you ran it and were the outputs are that most
users should just use without having to run this themselves.

Reply:
Done